src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,59 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 11;
+    //1 reserved for null, so limit set as 11 in case if none of the Keys is null
+    private static final int NULL_DEFAULT_INDEX = 0;
+    private int size = 0;
+    private int[] indexes;
+    private Object[] values;
```

Comment:
Let's use these fields
```suggestion
    private static final int MAX_STORAGE_SIZE = 10;
    private int size;
    private K[] keys;
    private V[] values;
```